ui/store/actions.test.js

```diff
@@ -660,7 +678,7 @@ describe('Actions', () => {
       const store = mockStore();
 
       const signMessage = background.signMessage.callsFake((_, cb) =>
-        cb(null, defaultState),
+        cb(null, defaultState.metamask),
```

Comment:
Ah yes! this must have been why when adding chainId lookup was failing my tests. removing this results in:
```
  ● Actions › #signMsg › calls signMsg in background

    TypeError: Cannot destructure property 'chainId' of 'state.metamask.provider' as it is undefined.

      55 |
      56 | export function getCurrentChainId(state) {
    > 57 |   const { chainId } = state.metamask.provider;
         |           ^
      58 |   return chainId;
      59 | }
```

because of selectors, we call in the new checks in the updateMetaMaskState. It probably worked before this as a consequence.